src/guides/v2.3/get-started/gs-curl.md

```diff
@@ -28,6 +28,128 @@ Option | Description
 `-T` | Transfers the specified local file to the remote URL.
 `-X` | Specifies the request method to use when communicating with the HTTP server. The specified request method is used instead of the default GET method.
 
+### Using cUrl in Magento
+
+Magento provides its own service-wrapper for using cURL instead of using default php cURL. The class ``Magento\Framework\HTTP\Client\Curl`` can be used to work with HTTP protocol using cURL library.
+At first, the instance of ``Magento\Framework\HTTP\Client\Curl`` should be created.
+
+```php
+/**
+* Constructor.
+*
+* @param Magento\Framework\HTTP\Client\Curl $curl
+*/
+public function __construct(
+   Magento\Framework\HTTP\Client\Curl $curl
+) {
+   $this->curl = $curl;
+}
+```
+#### Make GET request using cURL
+
+```php
+// get method
+$this->curl->get($url);
+
+// output of curl request
+$result = $this->curl->getBody();
+```
+
+Where the ``$url`` is the endpoint url
+
+#### Make POST request using cURL
+
+```php
+// post method
+$this->curl->post($url, $params);
+
+// output of curl requestt
+$result = $this->curl->getBody();
+```
+
+Where the ``$url`` is the endpoint url, ``$params`` is an array of data that is being sent via the POST request, the extra parameters can be added in the url.
```

Comment:
Since URL is an abbreviation, I would suggest using the upper case everywhere. The same suggestion applies to other places of this document.

```suggestion
Where the ``$url`` is the endpoint URL, ``$params`` is an array of data that is being sent via the POST request, the extra parameters can be added in the URL.
```